### PR TITLE
Add OpeningBrace Rule

### PR DESCRIPTION
**Pull request by S2dentik**
Added support for opening brace after to be located one space after declaration and on the same line.

### CODE REVIEW

File: Source/SwiftLintFramework/Configuration.swift [hunk 1]
```diff
@@ -152,13 +152,15 @@ public struct Configuration {
         } else {
             rules.append(TypeBodyLengthRule())
         }
-        if let params = yaml?[.String(FunctionBodyLengthRule().identifier)].arrayOfInts {
+        if let params   = yaml?[.String(FunctionBodyLengthRule().identifier)].arrayOfInts {
```
Comment (jpsim) [inline Source/SwiftLintFramework/Configuration.swift]: why was this extra space added?

**Comment by jpsim** [comment]
This looks good, @S2dentik! Could you please add an entry to the CHANGELOG.md and address the extra spacing that was added? Thanks!

**Comment by keith** [comment]
How does/should this affect the case where you have to put the opening bracket on another line because of the line length restrictions?

```swift
func foo...max...chars
{
```

It would probably be ok if it forced an arg on the next line?

**Comment by jpsim** [comment]
How do you suggest we codify that, @keith?

**Comment by keith** [comment]
That's a great question :smile: posing the question right now because that could generate unintended warnings if we're not handling it.

**Comment by S2dentik** [comment]
@keith, I guess line length rule should handle this, well not exactly this, it should leave the choice to the programmer on how to change the code based on the required style, but it shouldn't affect this rule if it is being checked, as it is superior.

**Pull request closed without merging**